doc/classes/GDExtension.xml

```diff
@@ -1,49 +1,66 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="GDExtension" inherits="Resource" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		A native library for GDExtension.
 	</brief_description>
 	<description>
+		The [GDExtension] resource type represents a [url=https://en.wikipedia.org/wiki/Shared_library]shared library[/url] written for GDExtension, which can expand the functionality of the engine. The [GDExtensionManager] singleton is responsible for loading, reloading, and unloading [GDExtension] resources.
```

Comment:
```suggestion
		The [GDExtension] resource type represents a [url=https://en.wikipedia.org/wiki/Shared_library]shared library[/url] which can expand the functionality of the engine. The [GDExtensionManager] singleton is responsible for loading, reloading, and unloading [GDExtension] resources.
```

Reply:
I would just say a GDExtension resource expands the functionality as GDExtension is the underlying system enabling this. It is not necessarily written **for** it.